services/discovery/template.yaml

```diff
@@ -16,8 +16,10 @@ Resources:
   DiscoveryFunction:
     Type: AWS::Serverless::Function 
     Properties:
-      CodeUri: src/
-      Handler: app.handler
+      CodeUri:
+        Bucket: !Sub 'cz-provision-account-${AWS::Region}'
```

Comment:
Is this our bucket?  Which account is it in?  We can only have one of these in each region without having name collisions.

Reply:
These are our buckets. They are in `cz-prime`. It sucks to have to create ~ 20 buckets; but it's the only way this will work (without inline CFN Lambdas ... [shudder]).

Reply:
OK, as long as we don't get collisions that will be fine 👍